15/umbraco-cms/reference/routing/iisrewriterules.md

```diff
@@ -114,7 +128,9 @@ Another example would be to enforce HTTPS only on your site:
 </rule>
 ```
 
-Another example would be to redirect from non-www to www (except for the Umbraco Cloud project hostname):
+### Example: Redirect Non-www to www
+
+To redirect traffic from non-www to www (excluding the Umbraco Cloud project hostname):
```

Comment:
Should the part about Cloud be added in a note? Otherwise it might be easy to miss 🤔 

```suggestion
The following rule redirects traffic from non-www to www (excluding the Umbraco Cloud project hostname):
```